api/src/test/java/org/openmrs/module/shr/odd/api/impl/test/OnDemandDocumentServiceImplTest.java

```diff
@@ -69,8 +69,6 @@ public void beforeTest() throws Exception
 		Context.getAdministrationService().setGlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_FALSE_CONCEPT, "1066");
         Context.getAdministrationService().setGlobalProperty("shr-cdahandler.cacheMappedConcepts", "false");
 		Context.getAdministrationService().saveGlobalProperty(saveDir);
-		GlobalProperty disableValidation = new GlobalProperty(OpenmrsConstants.GP_DISABLE_VALIDATION, "true");
-		Context.getAdministrationService().saveGlobalProperty(disableValidation);
```

Comment:
This GP doesn't exist in any version before 2.0, unless you're saying we should make 2.0 the minimum required version, then I can add it back but we wanted to supported lower versions since 2.0 isn't yet widely adopted. But I still think it's wrong to turn on off this setting in production, it's really intended for testing and for advance users who might just want to temporarily turn off validation may be to be able to import some data that they'll surely fix and then re-enable validation.